pkg/collector/metric/container_metric.go

```diff
@@ -108,7 +108,7 @@ func NewContainerMetrics(containerName, podName, podNamespace, containerID strin
 
 // ResetCurr reset all current value to 0
 func (c *ContainerMetrics) ResetDeltaValues() {
-	c.CurrProcesses = 0
+	// c.CurrProcesses = 0
```

Comment:
um.. will this cause some side effect e.g we want to set process to 0 at every round calculation... 
but this will not do that any more , e.g the process keep incresaeing?